src/lucky_migrator/tasks/db/redo.cr

```diff
@@ -0,0 +1,13 @@
+require "colorize"
+
+class Db::Redo < LuckyCli::Task
+  banner "Rollback then run the last migration"
+
+  def call
+    begin
+      LuckyMigrator::Runner.new.redo_last_migration
```

Comment:
Instead of adding a new method, I think you can do this: 

```crystal
Db::Rollback.call
Db::Migrate.call
```

Then you don't have to worry about error handing, and if either the `Db::Migrate` or `Db::Rollback` tasks change, this take will also get those changes. Does that make sense?

Reply:
Ah yes that makes sense